participant-manager/src/app/about/about.component.html

```diff
@@ -1,8 +1,25 @@
+<header>
```

Comment:
Should this have copyright header?

Reply:
Hi @jkrogers18, we include copyright headers in only SVG images, not in HTML and CSS files.